docs/pages/enterprise/getting-started.mdx

```diff
@@ -228,12 +228,23 @@ role only allows SSH logins as `root@host`.
 ```yaml
 kind: role
 metadata:
-  name: access
+  name: access-sample
 spec:
+  # This example defines a typical role. It allows listing all resources
+  # with typical developer credentials.
   allow:
-    logins:
-    - '{{internal.logins}}'
-    - root
+    logins: ['{{internal.logins}}', root]
```

Comment:
If this is a user's first introduction to a typical Teleport role, I think we should be explicit about what `internal.logins` does. Adding explanatory comments above other fields would also help.

Reply:
@ptgott expanded